server/controllers/notification.js

```diff
@@ -0,0 +1,50 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+// createNotification, readNotification, getAll, getUnread
+exports.createNotification = asyncHandler(async (req, res) => {
+    const notification = JSON.parse(req.body)
+    try{
```

Comment:
We don't need to use a try/catch block as the `asyncHandler` takes cares of error handling (forwards it to error middleware) in case of promise rejections

Ditto throughout!

Reply:
Removed in commit.